doc(release): Prepare release notes for MODULES 2210 (#2573)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.10.10

Release date: `August 31, 2023`

#### Bug fixes

> - [Editor] Fixed an issue that prevented the user from using the extended output in the widget "output".

Reviewer: ```suggestion
- [Editor] Fixed an issue that made it impossible to use the extended output in the "output" widget.
```

- [Server] Fixed an issue that prevented the MAP Engine server from starting.
- [Server] Updated dependencies.

> - [Viewer] Fixed the alignment of output text in the widget "output".

Reviewer: ```suggestion
- [Viewer] Fixed the alignment of output text in the "output" widget.
```
